### ISSUE TITLE
Remove field choices from model

### ISSUE THREAD

**Issue by KevinEtienne** [incuna/incuna-surveys#6]
https://github.com/incuna/incuna-surveys/pull/3#discussion_r73850582

> Maybe the choices can be set in the `form` or the `serializer` and keep `field_type` as a simple CharField?

**Comment by maxpeterson** [comment]
The choices will be different per question / field so I don't think they can be on the `form` or `serializer`

**Comment by KevinEtienne** [comment]
Yes good point, I was thinking about the admin form or the serializer to create the form.
